packages/astro/src/pages/about/code-of-conduct.md

```diff
@@ -0,0 +1,83 @@
+
+7. Encourage members and participants to listen as much as they speak.
+
+8. Strive to build tools that are open and free technology for public use. Activities that aim to foster public use, not for private gain, are prioritized.
+
+9. Prioritize access for and input from those who are traditionally excluded from the civic process.
+
+10. Work to ensure that all demographics present in the community are represented.
+
+11. Actively involve community groups and those with subject matter expertise in the decision-making process.
+
+12. Ensure that the relationships and conversations between community members, the local government staff and community partners remain respectful, participatory, and productive.
+
+13. Provide an environment where people are not subjected to discrimination or harassment. SF Civic Tech reserves the right to ask anyone in violation of these policies not to participate in SF Civic Tech events or network activities.
+
+
+## Anti-Harassment Policy
+
+#### What is harassment?
+Harassment is any unwanted behavior, physical or verbal (or even suggested ), that makes someone feel uncomfortable, humiliated, or mentally distressed. While harassment laws differ between states, most states consider these two main factors when deciding the validity of an accusation:
+
+* Behavior that annoy, threaten, or demean another - whether intentional or unintentional
+* Repetition and severity of the unwanted action.
+
+#### Some examples
+Harassment includes but is not limited to: offensive verbal or written comments related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, religion; sexual images in public spaces; deliberate intimidation; stalking; following; cyberbullying; cyberstalking; harassing photography or recording; sustained disruption of talks or other events; inappropriate physical contact; unwelcome sexual attention; unwarranted exclusion; and patronizing language or action.
+
+#### Our dedication to the community
+SF Civic Tech is dedicated to providing a harassment-free experience for everyone regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, or religion. We do not tolerate harassment of staff, presenters, and participants in any form. Sexual language and imagery are not appropriate for any SF Civic Tech event or network activity, including talks. Anyone in violation of these policies may be expelled from SF Civic Tech network activities, events, and digital forums, at the discretion of the event organizer or forum administrator.
+
+#### Repercussions for harassing someone in our community
+If a participant engages in harassing behavior, the organizers may take any action they deem appropriate, including warning the offender or expulsion from SF Civic Tech network activities, events, and digital forums.
+
+#### Speak up
```

Comment:
These should probably be h3s, since they're under an h2.  So `###` instead of `####`.

Reply:
My thought on adding the `####` instead of the `###` was more for stylistic purposes since the h3s were too large. This was before our discussion where we spoke about keeping these to .md files and instead editing the rendered html style. I have just pushed an update to these header tags.